scripts: sbom: Add "west sbom" tool. (#7209)
nrfconnect/sdk-nrf: doc/nrf/introduction.rst

If a folder or set of files is open source and included in |NCS| under its own license (for example, any of the Apache or MIT licenses), it will have either its own :file:`LICENSE` file included in the folder or the license information embedded inside the source files themselves.

> The west :ref:`ncs-sbom <west_sbom>` utility is used to generate a license report.

Reviewer: ```suggestion
You can use the west :ref:`ncs-sbom <west_sbom>` utility to generate a license report.
```

> It allows to generate report for the |NCS|, built application or for specific files.

Reviewer: ```suggestion
It allows you to generate a report for the |NCS|, built application, or specific files.
```

> The tool is configurable in a wide range.

Reviewer: ```suggestion
The tool is highly configurable.
```

> It uses several detection methods e.g. based on spdx tags, searching license information in files or using the `Scancode-Toolkit`_.

Reviewer: ```suggestion
It uses several detection methods, such as:

 * Search based on SPDX tags.
 * Search license information in files.
 * Use of the `Scancode-Toolkit`_.
```

> The report is generated in HTML and/or SPDX format.

Reviewer: Depending on your configuration, the report is generated in HTML or SPDX format, or in both formats.

> The report is generated in HTML and/or SPDX format.

Reviewer: ```suggestion
Depending on your configuration, the report is generated in HTML or SPDX format, or in both formats.
```
